source/NPC.h

```diff
@@ -56,9 +57,9 @@ class NPC {
 	~NPC() noexcept = default;
 
 	// Construct and Load() at the same time.
-	NPC(const DataNode &node);
+	NPC(const DataNode &node, std::string missionName = "");
```

Comment:
do we need a default argument? I feel like we always give this missionName, dont we?